about.html

```diff
@@ -49,7 +49,7 @@ <h2 style="text-align: left; padding-left: 1em">Interest Programs</h2>
   Interest Programs hold meetings separate from RITSEC’s general meeting to go
   over specific topics our members are passionate about. Interest Programs offer
   more events, have larger presences, and usually have membership requirements in order to join!
-  Reach out to the leaders of the interest programs if you have any questions! Our current IP's are as follows:
+  Reach out to the leaders of the interest programs if you have any questions! Our current Interest Programs are as follows:
```

Comment:
This is a test comment to show Olivia how PR's work

Reply:
hi